src/main/java/com/example/CustomerService/services/ContactDetailsService.java

```diff
@@ -0,0 +1,72 @@
+package com.example.CustomerService.services;
+
+import com.example.CustomerService.entity.ContactDetails;
+import com.example.CustomerService.entity.CustomerCredentials;
+import com.example.CustomerService.exception.CustomerNotFoundException;
+import com.example.CustomerService.model.CustomerContact;
+import com.example.CustomerService.repository.ContactDetailsRepository;
+import com.example.CustomerService.repository.CustomerCredetialsRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Service
+public class ContactDetailsService {
+    ContactDetailsRepository contactDetailsRepository;
+    CustomerCredetialsRepository customerCredetialsRepository;
+
+    ContactDetailsService(ContactDetailsRepository contactDetailsRepository, CustomerCredetialsRepository customerCredetialsRepository){
+        this.contactDetailsRepository=contactDetailsRepository;
+        this.customerCredetialsRepository=customerCredetialsRepository;
+    }
+
+    public void addContact(String name, CustomerContact customerContact) throws CustomerNotFoundException{
+        CustomerCredentials findCustomer=customerCredetialsRepository.findByUsername(customerContact.getUsername());
+//        if(customerContact.getUsername().equals(name)){
+//            log.info("enter valid credentials");
+//            throw new CustomerNotFoundException("give proper name for display contact details");
```

Comment:
I understand that you are commenting code to reference it in the future. But try to remove it if it is not needed before submitting :)